benchmarks/htap/queries/18.sql.template

```diff
@@ -26,7 +26,7 @@ group by
     o_entry_d,
     o_ol_cnt
 having
-    sum(ol_amount) > 200
+    sum(ol_amount) > 110000
```

Comment:
why that?

Reply:
to limit result size going to the python script. given how it runs this makes it just ~1k rows or so.